test.js

```diff
@@ -20,6 +19,7 @@ test('const chrome = nanochrome(url[, opts])', (t) => {
       t.notOk(err)
       chrome.close((err) => {
         t.notOk(err)
+        server.close()
```

Comment:
Moving this to here closes the server reliably, however, this does seem to indicate our chrome process is never quite making a connection to our server.  Problem with the test?

Reply:
maybe connecting to localhost in the CI server is an issue for headless chrome?

Reply:
ah i think we need to listen on `host` in the `listen()` call